cg/meta/orders/fastq_submitter.py

```diff
@@ -81,7 +81,7 @@ def store_items_in_status(
         if not customer:
             raise OrderError(f"Unknown customer: {customer_id}")
         new_samples = []
-        case: Family = self.status.find_family(customer=customer, name=ticket_id)
+        case: Family = self.status.get_case_by_name_and_customer(customer=customer, name=ticket_id)
```

Comment:
```suggestion
        case: Family = self.status.get_case_by_name_and_customer(customer=customer, case_name=ticket_id)
```
It is not clear that (at least to me) the ticket id is the same as the case name

Reply:
Me neither, but that is how the function was implemented. Looking into it

Reply:
In some places, the name of the case is set as the ticket id or some variation on the ticket id (for instance by adding a suffix). In other places, the name of the case does not seem to be explicitly related to the ticket id. I created a separate issue for it https://github.com/Clinical-Genomics/cg/issues/1862